src/components/RegistrationForm.tsx

```diff
@@ -154,7 +158,16 @@ export default function RegistrationForm({ handles, wardens, className }) {
           setHasValidationErrors(true);
           return;
         }
-        // @todo: validate discord handle
+
+        if (isDangerousHandle(state.username)) {
+          setHasValidationErrors(true);
+          setStatus(FormStatus.Error);
+          updateErrorMessage(
+            "Handle can only contain alphanumeric characters, underscores, and hyphens"
+          );
```

Comment:
Let's do inline validation for this instead. You can follow the example of how you implemented validating discord usernames.

Reply:
I just did it :) and add :

```

              {hasValidationErrors &&
                state.teamName &&
                isDangerousTeamName(state.teamName) && (
                  <p className={widgetStyles.ErrorMessage}>
                    <small>
                      Supports alphanumeric characters, underscores, and hyphens
                    </small>
                  </p>
                )}
```
                
as an error